File: 프로그래머스/0/120837. 개미 군단/개미 군단.py
def solution(hp):
    a = hp // 5
    a1 = hp % 5
    b = a1 // 3
    c = a1 % 3
    
    return a + b + c

print(solution(23))     # 결과값 : 5
print(solution(24))     # 결과값 : 6
print(solution(999))    # 결과값 : 201

File: 프로그래머스/0/120837. 개미 군단/개미 군단.js
function solution(hp) {
    let reminder = hp % 5;
    let general = (hp - reminder) / 5;
    let worker = reminder % 3;
    let soldier = (reminder - worker) / 3;
    return general + soldier + worker;
}

console.log(solution(23));
console.log(solution(24));
console.log(solution(999));